src/System.CommandLine.Generator/WellKnownTypes.cs

```diff
@@ -19,7 +19,7 @@ internal class WellKnownTypes
         public WellKnownTypes(Compilation compilation, IEqualityComparer<ISymbol?> comparer)
         {
             Console = GetType("System.CommandLine.IConsole");
-            ParseResult = GetType("System.CommandLine.Parsing.ParseResult");
+            ParseResult = GetType("System.CommandLine.ParseResult");
```

Comment:
```suggestion
            ParseResult = GetType(nameof(System.CommandLine) + '.' + nameof(ParseResult));
```
@jonsequitur is there a reason here we don't use nameof() like this? Would make any breaking changes of namespaces easier to modify as it took me some time to find that this caused test failures because the namespace move didn't adjust these hardcoded strings

Reply:
This is a source generator. It doesn't have a dependency on System.CommandLine so this won't compile.

There might be clever approaches to this kind of think but it's not something I've explored much. @KathleenDollard might have some insights here.

Reply:
Totally right, I forgot to look up at the fact that it is a source generator, just jumped to the hardcoded line I remember I had to modify. I would be interested if it is possible, could be interesting!